### DISCUSSION TITLE
No packages found.

### DISCUSSION THREAD

**Discussion by yefengwu** [thilojaeggi/WinGetty#38] [Q&A]
[2023-12-16 13:41:24 +0100] [9] [INFO] [WinGetty] No packages found.

**Comment by yefengwu** [comment]
When I use "search," it can find the package, but when I use "install" or "show," it says "no package found."

**Reply by thilojaeggi** [reply]
Did you get it fixed? What was the issue?

**Reply by yefengwu** [reply]
This shouldn't be a problem, just add --id when installing the program
